src/omnisharp/protocol.ts

```diff
@@ -623,6 +624,18 @@ export namespace V2 {
         MessageLevel: string;
         Message: string;
     }
+
+    export interface BlockStructureResponse
+    {
+        Spans: BlockSpan[]
+    }
+
+    export interface BlockSpan {
+        TextSpan: Range;
+        HintSpan: Range;
+        BannerText: string;
+        Type: string;
+    }
```

Comment:
This protocol should match other OmniSharp endpoints better. It'd probably better to have just a single `Range` property rather than all of the other properties that are ultimately unused.

Reply:
`BlockStructureRequest` appears to be missing

Reply:
@DustinCampbell This is something the extension codebase is inconsistent about: for request types that derive Request but don't add any properties, we haven't been adding corresponding entries in protocol.ts (instead just preferring to use Request directly) (case in point: there's no FileMembersAsTree request type in the extension). I was following that pattern here. We could also switch to adding request-deriving types that don't add properties. What do you think?